### ISSUE TITLE
Superblock: Notifications for superblocked users

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5914]
### Expected behavior
When I superblock a user I don't expect to get notifications about comments from the superblocked user.

### Actual behavior
Whenever a superblocked user is commenting in a thread I'm involved in I get a notification about the comment. When I click on the notification I'm redirected to to thread and (of cause) there is no new comment (because it has been blocked).

### Steps to reproduce the problem
Difficult. :-) Add a user to the superblock list. Comment on some threads and wait for the notifications.

### Friendica version you encountered the problem
2018.12-dev

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
10.1.34 MariaDB

**Comment by annando** [comment]
The reason is clear, superblock is some addon and has limited possibilities. Best is to replace this with the new ways we added to block someone.

**Comment by annando** [comment]
You can unblock them on the same way. Means: You can search for the contact in the search.

**Comment by AlfredSK** [comment]
Oh! The search! 🤦 I thought I cannot unblock the user because I will never see him/her again because of the blocked state. :-)

**Comment by WNYmathGuy** [comment]
Instead of creating a new issue, I'll just tell my _bug-like_ experience here...

I was trying to right click then click an option on my screen from muscle memory, but an OS thing stalled and that delayed mouse movement instead right clicked my own profile image then clicked something in the list. I couldn't see any of my own posts anymore. I was baffled. I looked in my contacts list to see if I blocked myself, but I wasn't there.

Digging in the database I saw my `uid` in the `permissionset` table with a blocking `allowcid` entry of `<832>`. I didn't know what the hell that meant, but I dug into the "Superblock" add-on settings and found myself in the `Comma-separated profile URLs to block` section. Turns out the `permissionset` table had nothing to do with it. Just wandered lucky into a solution before opening a new Issue here.

Maybe this note will help some new newbie like me.
